docs/cpp-sdk-factory-design.md

```diff
@@ -94,6 +94,45 @@ This property makes it possible to:
 - deploy a new SDK shared library
 - keep the application unchanged
 
+### Case study, using Factory and shared gRPC client between OTLP gRPC exporters
+
+To reduce the cost of gRPC, the SDK allow users to shared gRPC clients between
+OTLP gRPC exporters when these exporters have the same settings. This can be
+used as follows from the application code:
+
+```cpp
+// Include follows headers
+#include "opentelemetry/exporters/otlp/otlp_grpc_client_factory.h"
+#include "opentelemetry/exporters/otlp/otlp_grpc_exporter_factory.h"
+#include "opentelemetry/exporters/otlp/otlp_grpc_exporter_options.h"
+#include "opentelemetry/exporters/otlp/otlp_grpc_log_record_exporter_factory.h"
+#include "opentelemetry/exporters/otlp/otlp_grpc_log_record_exporter_options.h"
+
+// Create exporters with shared gRPC Client
+namespace otlp = opentelemetry::exporter::otlp;
+
+void SetupOtlp() {
+  otlp::OtlpGrpcExporterOptions trace_opts;
+  otlp::OtlpGrpcLogRecordExporterOptions log_opts;
+
+  // Setting trace_opts and log_opts
+  std::shared_ptr<otlp::OtlpGrpcClient> shared_client =
+    otlp::OtlpGrpcClientFactory::Create(trace_opts);
```

Comment:
For clarity, maybe use:

```
OtlpGrpcClientOptions client_opts;
```

instead of trace_opts, and call `OtlpGrpcClientFactory::Create(client_opts)`.

Reply:
Thanks and updated.